src/main/resources/swagger/api-docs.yaml

```diff
@@ -357,15 +357,11 @@ definitions:
     description: ''
     required:
       - bucketPath
-      - serviceAccount
```

Comment:
heads up - this will probably break our demo ui

Reply:
Yes! I meant to bring this up -- this PR will totally break the demo we've been doing thus far, because we'll no longer be using the special leo-demo account, but instead the user permissions in dev, which probably won't work calling back to FC prod. Maybe I'll follow up with Brad on this -- is it OK to temporarily break the demo until we get to prod, or should we build some kind of backdoor where we can still use the leo-demo credentials?

Reply:
Brad said ok to break the demo until we go to prod

Reply:
Yeah - as long as we don't have to give the demo again for a while - should be fine